cmd-programs.muf

```diff
@@ -18,13 +18,20 @@ $include $lib/strings
     "FV" var! vflags  ( strX )
 
     dup if  ( strX )
-        "mine" stringcmp if
-            me @ vowner !
-            "F" vflags !
-        then  (  )
+        case
+            "help" stringcmp not when .showhelp exit end
+            "mine" stringcmp not when
+                me @ vowner !
+                "F" vflags !
+            end
+            default
+              "I don't know what you mean by '#%s'. 'programs #help' for help." fmtstring tell  (  )
+              exit
+            end
+        endcase
     else pop then  (  )
 
-    " Dbref Name                     Author               Owner      Modified @view" "bold" textattr .tell
+    " Dbref Name                     Author               Owner      Modified @view" "bold" textattr tell
```

Comment:
While other programs here still use the `.tell` macro, we can guess that the `tell` primitive will be more commonly available.